fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -296,10 +296,14 @@ public void removeMixin(final FedoraResource resource,
                             final Resource mixinResource,
                             final Map<String, String> nsPrefixMap) throws RepositoryException {
 
-        final Node node = resource.getNode();
-        final String mixinName = getPropertyNameFromPredicate(node, mixinResource, nsPrefixMap);
-        if (repositoryHasType(session, mixinName) && node.isNodeType(mixinName)) {
-            node.removeMixin(mixinName);
+        try {
```

Comment:
Since `JcrRdfTools.removeMixin()` already throws a `RepositoyException`, the `try|catch` you have added can be safely removed.